statistics/handle_test.go

```diff
@@ -35,16 +36,42 @@ func TestT(t *testing.T) {
 
 var _ = Suite(&testStatsCacheSuite{})
 
-type testStatsCacheSuite struct{}
+type testStatsCacheSuite struct {
+	store kv.Storage
+	do    *domain.Domain
+}
 
-func (s *testStatsCacheSuite) TestStatsCache(c *C) {
-	store, do, err := newStoreWithBootstrap()
+func (s *testStatsCacheSuite) SetUpSuite(c *C) {
+	var err error
+	s.store, s.do, err = newStoreWithBootstrap()
 	c.Assert(err, IsNil)
-	defer store.Close()
-	testKit := testkit.NewTestKit(c, store)
+}
+
+func (s *testStatsCacheSuite) TearDownSuite(c *C) {
+	s.store.Close()
+}
+
+func (s *testStatsCacheSuite) cleanEnv(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
```

Comment:
This code is repeated three times. You can use a function  named cleanStatsEnv( c *C, do *domain.Domain, store kv.Storage) to do this work.